components/footer/HelpChat.js

```diff
@@ -8,64 +8,57 @@
  */
 
 // dependencies
+import { useState } from 'react';
 import { EmojiLaughing, CaretLeftFill } from 'react-bootstrap-icons';
 // local files
+import styles from './HelpChat.module.css';
+import ChatWindow from './ChatWindow';
 
 /**
- * @description - Help Chat icon and button that opens a chat window on click
+ * @description - Help Chat icon. Opens a chat window on click
+ * @param {string} props.helpChatText - chat text to display on screens > small (576px)
+ * @param {string} props.helpChatTextSmallScreen - chat text to display on screens < small (576px)
  * @return {jsx} - the HelpChat component to render
  */
-const HelpChat = () => {
+const HelpChat = ({ helpChatText, helpChatTextSmallScreen }) => {
+  // state
+  const [open, setOpen] = useState(false);
+
+  // event handlers
+  /**
+   * @description - on click event listener that toggles "open" state between true or false boolean values
+   * @return {undefined} - Nothing to return
+   */
+  const toggleChatWindowDisplay = () => {
+    setOpen(!open);
+  };
+
+  // conditional renders
+  const renderChatWindow = open ? (
```

Comment:
since we only want to render the ChatWindow component, we can use the `&&` operator instead of the ternary if/else operator. [React Conditional Rendering Documentation](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)
```
const renderChatWindow = open && <ChatWindow/>;
```

Reply:
is this a function or a variable? since the name is `renderChatWindow` (starts with a verb), it makes it seem like it is a function - in which case, we need a function description. happy to chat about it if this needs clarification

Reply:
function or variable: My mistake - function declaration/expressions start with verbs, variables are nouns. Supposed to be a variable, so will change to a noun. Thanks!

Reply:
@achang-tcs Ready for review! Note* The variable name is too similar to the component e.g. chatWindow & ChatWindow (component), and was thinking of getting rid of & placing the variable's contents inside the return statement. However, I heard to keep component returns as simple as possible (move all logic into separate helper functions/variables). Wondering what is your view on this? Thanks!

Reply:
@achang-tcs Ready for review! Decided to go for helper function approach to keep return statement nice and tidy (personal pref!)